### ISSUE TITLE
Some pali matches don't show up

### ISSUE THREAD

**Issue by ayya-vimala** [BuddhaNexus/buddhanexus-frontend#245]
When I go to DN 1 and click on the text "antarā ca rājagahaṁ antarā ca nāḷandaṁ" or thereabouts I get 7 matches. setting the filters full open I get 15. But when I search for the exact sentence in Sublime, I get some matches that are exact matches that do not appear in the results.

The exact matches I got in other files than DN1 are only 3:
```
"anya-e0201n2:73": "Dhī brāhmaṇassa hantāraṁ dha. pa. 389, dhīratthu’maṁ pūtikāyaṁ jā. 1.3.129, dhīratthu taṁ dhanalābhaṁ jā. 1.4.36, dhīratthu bahuke kāme. Tatiyāpi dissati, dhīratthu jīvitena me jā. 2.17.135. Antarā ca rājagahaṁ antarā ca nāḷandaṁ dī. ni. 1.1, abhito gāmaṁ vasati, paritogāmaṁ vasati, nadiṁ nerañjaraṁ pati su. ni. 427, etesu chaṭṭhyatthe dutiyā.",
"anya-e0805n2:63": "Chaṭṭhīnaṁ atthe kvaci dutiyāvibhatti hoti. Antarāabhito parito pati paṭibhātiyoge ayaṁ. Antarā ca rājagahaṁ antarā ca nāḷandaṁ addhānamaggappaṭipanno, rājagahassa ca nāḷandāya ca majjheti attho. Abhito gāmaṁ vasati, parito gāmaṁ vasati, nadiṁ nerañjaraṁ pati, nerañjarāya nadiyā samīpeti attho. Paṭibhantu taṁ cunda bojjhaṅgā, upamā maṁ paṭibhāti, upamā mayhaṁ upaṭṭhahatīti attho.",
"sn16:11.9.1": "So evaṁ pabbajito samāno addhānamaggappaṭipanno addasaṁ bhagavantaṁ antarā ca rājagahaṁ antarā ca nāḷandaṁ bahuputte cetiye nisinnaṁ. ",
```
But the first one and the last one are not in the results on BN

**Comment by ayya-vimala** [comment]
After discussing with Ven. Anandajoti, he also indicated the following:
```
"anya-e0901n1:2": "Vissajjanā – antarā ca bhante rājagahaṁ antarā ca nāḷandaṁ ambalaṭṭhikāyaṁ rājāgārake bhāsitaṁ.",
"atk-s0101a0:99": "Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ etadavoca, ‘‘brahmajālaṁ, āvuso ānanda, kattha bhāsita’’nti? ‘‘Antarā ca, bhante, rājagahaṁ antarā ca nāḷandaṁ rājāgārake ambalaṭṭhikāya’’nti. ‘‘Kaṁ ārabbhā’’ti ? ‘‘Suppiyañca paribbājakaṁ, brahmadattañca māṇava’’nti. ‘‘Kismiṁ vatthusmi’’nti? ‘‘Vaṇṇāvaṇṇe’’ti. Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ brahmajālassa nidānampi pucchi, puggalampi pucchi, vatthumpi pucchi (cūḷava. 440). Āyasmā ānando vissajjesi. Vissajjanāvasāne pañca arahantasatāni gaṇasajjhāyamakaṁsu. Vuttanayeneva ca pathavikampo ahosi.",
"atk-s0501a5:49": "Atha kho āyasmā ānando uṭṭhāyāsanā ekaṁsaṁ cīvaraṁ katvā there bhikkhū vanditvā dhammāsane nisīdi dantakhacitaṁ bījaniṁ gahetvā. Atha mahākassapatthero ānandattheraṁ dhammaṁ pucchi – ‘‘brahmajālaṁ, āvuso ānanda, kattha bhāsita’’nti? ‘‘Antarā ca, bhante, rājagahaṁ antarā ca nāḷandaṁ rājāgārake ambalaṭṭhikāya’’nti. ‘‘Kaṁ ārabbhā’’ti ? ‘‘Suppiyañca paribbājakaṁ brahmadattañca māṇavaka’’nti. Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ brahmajālassa nidānampi pucchi, puggalampi pucchi. ‘‘Sāmaññaphalaṁ; panāvuso ānanda, kattha bhāsita’’nti? ‘‘Rājagahe, bhante, jīvakambavane’’ti. ‘‘Kena saddhi’’nti? ‘‘Ajātasattunā vedehiputtena saddhi’’nti. Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ sāmaññaphalassa nidānampi pucchi, puggalampi pucchi. Eteneva upāyena pañcapi nikāye pucchi, puṭṭho puṭṭho āyasmā ānando vissajjesi. Ayaṁ paṭhamamahāsaṅgīti pañcahi therasatehi katā –",
"atk-vin01a0:116": "Atha kho āyasmā ānando uṭṭhāyāsanā ekaṁsaṁ cīvaraṁ katvā there bhikkhū vanditvā dhammāsane nisīdi dantakhacitaṁ bījaniṁ gahetvā. Atha mahākassapatthero ānandattheraṁ dhammaṁ pucchi – ‘‘brahmajālaṁ, āvuso ānanda, kattha bhāsita’’nti? ‘‘Antarā ca, bhante, rājagahaṁ antarā ca nāḷandaṁ rājāgārake ambalaṭṭhikāya’’nti. ‘‘Kaṁ ārabbhā’’ti? ‘‘Suppiyañca paribbājakaṁ, brahmadattañca māṇava’’nti. ‘‘Kismiṁ vatthusmi’’nti? ‘‘Vaṇṇāvaṇṇe’’ti. Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ brahmajālassa nidānampi pucchi, puggalampi pucchi, vatthumpi pucchi. ‘‘Sāmaññaphalaṁ panāvuso ānanda, kattha bhāsita’’nti? ‘Rājagahe, bhante, jīvakambavane’’ti. ‘‘Kena saddhi’’nti? ‘‘Ajātasattunā vedehiputtena saddhi’’nti. Atha kho āyasmā mahākassapo āyasmantaṁ ānandaṁ sāmaññaphalassa nidānampi pucchi, puggalampi pucchi. Eteneva upāyena pañca nikāye pucchi.",
"pli-tv-kd21:1.8.9": "“Antarā ca, bhante, rājagahaṁ antarā ca nāḷandaṁ rājāgārake ambalaṭṭhikāyā”ti.",
 "anya-e1201n:147": "Dhiādīhi yuttato dutiyā hoti? Dhiratthumaṁ pūtikāyaṁ, antarā ca rājagahaṁ, antarā ca nāḷandaṁ, samādhānamantarena, mucalindamabhito saramiccādi-chaṭṭhiyāpavādo yaṁ.", But I checked in the tokanized files and that comma was removed in the tokanized version so that should not be it.

To summarize, what I can find is the following:
original text: `antarā ca rājagahaṁ antarā ca nāḷandaṁ`
alternatives: `antarā ca rājagahaṁ, antarā ca nāḷandaṁ`
`antarā ca rājagahaṁ antarā ca nālandaṁ` (missing a dot under the l)
`antarā ca, bhante, rājagahaṁ antarā ca nāḷandaṁ`
`antarā ca bhante rājagahaṁ antarā ca nāḷandaṁ`


```
So note that the word "bhante" is in between in the sentence. This is indeed a little stopword that might throw the system out a bit.

Also note that all the numbers I give you here seem to be very long segments but in the tokanized files, these are split up so in reality they are not that long.
Als a comma in `antarā ca rājagahaṁ, antarā ca nāḷandaṁ` seems to throw the system off but this might be because

**Comment by sebastian-nehrdich** [comment]
As I mentioned on github, the min. length for a match for the pali is currently at 50 chars. so That explains why we loose a lot of these examples since they are shorter than that (only about 38 chars). 
I can reduce that to 30 chars and have a look where we end up with that. The shorter the min. length the more results we get, of course.

**Comment by ayya-vimala** [comment]
Ah, is that so. I forgot. I thought the co-occurances were capped but it was the length then? Maybe that should be made clear instead (as now it says on the site that the co-occurances are capped). Also the filter shows '7' or so and no more so the user cannot find any reason as to why it would not show up.
I think maybe I will talk to Anandabodhi first and see what he thinks before you do another big run.
